lib/app/routes/app_router.dart

```diff
@@ -855,10 +867,6 @@ class AppRouter {
       return Pages.generosityChallenge.path;
```

Comment:
if (GenerosityChallengeHelper.isActivated || navigatingPage == Pages.generosityChallenge.path)  - it's important for when users launch the app from link

Reply:
fixed